2022/11/27 23:09:37 [emerg] 855#855: open() "/etc/nginx/sites-enabled/mysite_nginx.conf" failed (2: No such file or directory) in /etc/nginx/nginx.conf:62
2022/11/27 23:13:33 [emerg] 1564#1564: open() "/etc/nginx/sites-enabled/mysite_nginx.conf" failed (2: No such file or directory) in /etc/nginx/nginx.conf:62
2022/11/27 23:15:08 [emerg] 1840#1840: open() "/etc/nginx/sites-enabled/mysite_nginx.conf" failed (2: No such file or directory) in /etc/nginx/nginx.conf:62
2022/11/27 23:17:42 [emerg] 2181#2181: open() "/etc/nginx/sites-enabled/mysite_nginx.conf" failed (2: No such file or directory) in /etc/nginx/nginx.conf:62
2022/11/27 23:17:57 [emerg] 2257#2257: open() "/etc/nginx/sites-enabled/mysite_nginx.conf" failed (2: No such file or directory) in /etc/nginx/nginx.conf:62
2022/11/27 23:18:18 [emerg] 2378#2378: open() "/etc/nginx/sites-enabled/mysite_nginx.conf" failed (2: No such file or directory) in /etc/nginx/nginx.conf:62
2022/11/27 23:23:03 [emerg] 3437#3437: open() "/etc/nginx/sites-enabled/mysite_nginx.conf" failed (2: No such file or directory) in /etc/nginx/nginx.conf:62
2022/11/27 23:24:25 [emerg] 3605#3605: open() "/etc/nginx/sites-enabled/mysite_nginx.conf" failed (2: No such file or directory) in /etc/nginx/nginx.conf:62
2022/11/27 23:25:41 [error] 3763#3763: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "uwsgi://127.0.0.1:8001", host: "127.0.0.1:8000"
2022/11/27 23:25:41 [error] 3763#3763: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://127.0.0.1:8001", host: "127.0.0.1:8000", referrer: "http://127.0.0.1:8000/"
2022/11/27 23:26:41 [emerg] 3930#3930: open() "/etc/nginx/sites-enabled/mysite_nginx.conf" failed (2: No such file or directory) in /etc/nginx/nginx.conf:62
2022/11/27 23:26:51 [emerg] 3950#3950: open() "/etc/nginx/sites-enabled/mysite_nginx.conf" failed (2: No such file or directory) in /etc/nginx/nginx.conf:62
2022/11/27 23:27:47 [emerg] 4037#4037: open() "/etc/nginx/sites-enabled/mysite_nginx.conf" failed (2: No such file or directory) in /etc/nginx/nginx.conf:62
2022/11/27 23:34:01 [error] 4773#4773: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://127.0.0.1:8001", host: "127.0.0.1:8000", referrer: "http://127.0.0.1:8000/media/media.png"
2022/11/27 23:36:52 [error] 4773#4773: *3 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /runs/4c635a8e-28b5-4aa5-9811-d4aa3cb3431d HTTP/1.1", upstream: "uwsgi://127.0.0.1:8001", host: "127.0.0.1:8000", referrer: "http://localhost:3000/"
2022/11/27 23:36:53 [error] 4773#4773: *3 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /runs/4c635a8e-28b5-4aa5-9811-d4aa3cb3431d HTTP/1.1", upstream: "uwsgi://127.0.0.1:8001", host: "127.0.0.1:8000", referrer: "http://localhost:3000/"
2022/11/29 09:46:19 [emerg] 1856#1856: too long path in the unix domain socket in upstream "unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/barcode_identifier_api.sock" in /etc/nginx/sites-enabled/barcode_identifier_api_nginx.conf:3
2022/11/29 09:46:27 [emerg] 1904#1904: too long path in the unix domain socket in upstream "unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/barcode_identifier_api.sock" in /etc/nginx/sites-enabled/barcode_identifier_api_nginx.conf:3
2022/11/29 09:48:57 [emerg] 2036#2036: too long path in the unix domain socket in upstream "unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/barcode_identifier_api.sock" in /etc/nginx/sites-enabled/barcode_identifier_api_nginx.conf:3
2022/11/29 09:51:32 [crit] 2306#2306: *1 connect() to unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock failed (13: Permission denied) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "localhost:8000"
2022/11/29 09:51:33 [crit] 2306#2306: *1 connect() to unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock failed (13: Permission denied) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "localhost:8000", referrer: "http://localhost:8000/"
2022/11/29 09:51:43 [crit] 2306#2306: *4 connect() to unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock failed (13: Permission denied) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1:8000"
2022/11/29 09:51:43 [crit] 2306#2306: *4 connect() to unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock failed (13: Permission denied) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1:8000", referrer: "http://127.0.0.1:8000/"
2022/11/29 09:51:49 [crit] 2306#2306: *4 connect() to unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock failed (13: Permission denied) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1:8000"
2022/11/29 09:51:49 [crit] 2306#2306: *4 connect() to unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock failed (13: Permission denied) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1:8000", referrer: "http://127.0.0.1:8000/"
2022/12/22 11:17:01 [emerg] 1852#1852: too long path in the unix domain socket in upstream "unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/barcode_identifier_api.sock" in /etc/nginx/sites-enabled/barcode_identifier_api_nginx.conf:3
2022/12/22 11:18:27 [error] 2010#2010: *3 connect() to unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /nuccores HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1"
2022/12/22 11:18:27 [error] 2010#2010: *3 connect() to unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1", referrer: "https://127.0.0.1/nuccores"
2022/12/22 11:19:24 [error] 2010#2010: *3 connect() to unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /nuccores/ HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1"
2022/12/22 11:19:51 [error] 2010#2010: *3 connect() to unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /nuccores/ HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1"
2022/12/22 11:20:51 [error] 2010#2010: *3 connect() to unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1", referrer: "https://127.0.0.1/media/media.txt"
2022/12/22 12:33:22 [error] 2010#2010: *45 connect() to unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /blastdbs/4f33c746-e566-4cfb-a79d-1d4bcb8cae6d/add/ HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1"
2022/12/22 12:33:26 [error] 2010#2010: *45 connect() to unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /8000/blastdbs/4f33c746-e566-4cfb-a79d-1d4bcb8cae6d/add/ HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1"
2022/12/22 12:42:21 [error] 2010#2010: *48 connect() to unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /blastdbs/ HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1", referrer: "http://localhost:3000/"
2022/12/22 12:45:50 [error] 2010#2010: *50 connect() to unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /blastdbs/ HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1", referrer: "http://localhost:3000/"
2022/12/22 13:04:34 [error] 2010#2010: *52 connect() to unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /blastdbs/ HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1", referrer: "http://localhost:3000/"
2022/12/22 13:04:36 [error] 2010#2010: *52 connect() to unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /blastdbs/ HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1", referrer: "http://localhost:3000/"
2022/12/22 13:04:36 [error] 2010#2010: *52 connect() to unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /blastdbs/ HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1", referrer: "http://localhost:3000/"
2022/12/22 13:04:37 [error] 2010#2010: *52 connect() to unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /blastdbs/ HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1", referrer: "http://localhost:3000/"
2022/12/22 13:04:40 [error] 2010#2010: *52 connect() to unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /blastdbs/ HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1", referrer: "http://localhost:3000/"
2022/12/22 13:04:42 [error] 2010#2010: *52 connect() to unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /blastdbs/ HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1", referrer: "http://localhost:3000/"
2022/12/22 13:04:44 [error] 2010#2010: *52 connect() to unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /blastdbs/ HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1", referrer: "http://localhost:3000/"
2022/12/22 13:04:45 [error] 2010#2010: *52 connect() to unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /blastdbs/ HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1", referrer: "http://localhost:3000/"
2022/12/22 13:04:53 [error] 2010#2010: *52 connect() to unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /blastdbs/ HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1", referrer: "http://localhost:3000/"
2022/12/22 13:04:59 [error] 2010#2010: *52 connect() to unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /blastdbs/ HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1", referrer: "http://localhost:3000/"
2022/12/22 13:05:00 [error] 2010#2010: *52 connect() to unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /blastdbs/ HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1", referrer: "http://localhost:3000/"
2022/12/22 13:05:02 [error] 2010#2010: *52 connect() to unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /blastdbs/ HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1", referrer: "http://localhost:3000/"
2022/12/22 13:05:03 [error] 2010#2010: *52 connect() to unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /blastdbs/ HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1", referrer: "http://localhost:3000/"
2022/12/22 13:05:19 [error] 2010#2010: *52 connect() to unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /blastdbs/ HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1", referrer: "http://localhost:3000/"
2022/12/22 13:05:21 [error] 2010#2010: *52 connect() to unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /blastdbs/ HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1", referrer: "http://localhost:3000/"
2023/02/26 17:44:09 [error] 3299#3299: *3 connect() to unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1"
2023/02/26 17:44:09 [error] 3299#3299: *3 connect() to unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2023/02/26 17:44:12 [error] 3299#3299: *3 connect() to unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /nuccores HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1"
2023/02/26 18:25:06 [error] 3299#3299: *58 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /blastdbs/ef2e0deb-6105-4bfe-b4f9-578928150576/run HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1"
2023/02/26 18:27:45 [error] 3299#3299: *60 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /blastdbs/ef2e0deb-6105-4bfe-b4f9-578928150576/run HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1"
2023/02/26 19:09:16 [error] 3299#3299: *79 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/style.css" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/style.css HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/26 19:09:16 [error] 3299#3299: *79 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/swagger-ui-init.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/swagger-ui-init.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/26 19:09:16 [error] 3299#3299: *84 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/swagger-ui-dist/swagger-ui.css" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/swagger-ui-dist/swagger-ui.css HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/26 19:09:16 [error] 3299#3299: *85 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/swagger-ui-dist/swagger-ui-bundle.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/swagger-ui-dist/swagger-ui-bundle.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/26 19:09:16 [error] 3299#3299: *86 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/swagger-ui-dist/swagger-ui-standalone-preset.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/swagger-ui-dist/swagger-ui-standalone-preset.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/26 19:09:16 [error] 3299#3299: *87 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/immutable.min.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/immutable.min.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/26 19:09:16 [error] 3299#3299: *88 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/insQ.min.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/insQ.min.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/26 19:09:16 [error] 3299#3299: *79 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/immutable.min.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/immutable.min.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/26 19:09:16 [error] 3299#3299: *79 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/swagger-ui-init.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/swagger-ui-init.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/26 19:09:16 [error] 3299#3299: *79 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/swagger-ui-dist/favicon-32x32.png" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/swagger-ui-dist/favicon-32x32.png HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/26 19:09:36 [error] 3299#3299: *79 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/style.css" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/style.css HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/26 19:09:36 [error] 3299#3299: *84 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/swagger-ui-dist/swagger-ui-bundle.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/swagger-ui-dist/swagger-ui-bundle.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/26 19:09:36 [error] 3299#3299: *85 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/swagger-ui-dist/swagger-ui-standalone-preset.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/swagger-ui-dist/swagger-ui-standalone-preset.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/26 19:09:36 [error] 3299#3299: *79 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/swagger-ui-dist/swagger-ui.css" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/swagger-ui-dist/swagger-ui.css HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/26 19:09:36 [error] 3299#3299: *86 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/insQ.min.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/insQ.min.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/26 19:09:36 [error] 3299#3299: *88 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/swagger-ui-init.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/swagger-ui-init.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/26 19:09:36 [error] 3299#3299: *87 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/immutable.min.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/immutable.min.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/26 19:09:36 [error] 3299#3299: *79 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/insQ.min.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/insQ.min.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/26 19:09:36 [error] 3299#3299: *79 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/immutable.min.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/immutable.min.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/26 19:09:36 [error] 3299#3299: *79 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/swagger-ui-init.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/swagger-ui-init.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/26 19:14:55 [error] 3299#3299: *91 connect() to unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /swagger.yaml HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1"
2023/02/26 19:18:49 [error] 3299#3299: *96 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /swagger.yaml HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1", referrer: "http://localhost:3000/"
2023/02/26 19:18:49 [error] 3299#3299: *97 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /swagger.yaml HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1", referrer: "http://localhost:3000/"
2023/02/26 20:27:47 [error] 3299#3299: *118 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/style.css" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/style.css HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/26 20:27:47 [error] 3299#3299: *118 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/immutable.min.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/immutable.min.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/26 20:27:47 [error] 3299#3299: *122 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/insQ.min.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/insQ.min.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/26 20:27:47 [error] 3299#3299: *121 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/swagger-ui-dist/swagger-ui-standalone-preset.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/swagger-ui-dist/swagger-ui-standalone-preset.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/26 20:27:47 [error] 3299#3299: *123 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/swagger-ui-dist/swagger-ui.css" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/swagger-ui-dist/swagger-ui.css HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/26 20:27:47 [error] 3299#3299: *120 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/swagger-ui-dist/swagger-ui-bundle.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/swagger-ui-dist/swagger-ui-bundle.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/26 20:27:47 [error] 3299#3299: *124 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/swagger-ui-init.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/swagger-ui-init.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/26 20:27:47 [error] 3299#3299: *118 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/swagger-ui-dist/favicon-32x32.png" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/swagger-ui-dist/favicon-32x32.png HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/26 21:53:40 [error] 3299#3299: *136 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /swagger.yaml HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1", referrer: "http://localhost:3000/"
2023/02/26 21:53:40 [error] 3299#3299: *139 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /swagger.yaml HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1", referrer: "http://localhost:3000/"
2023/02/26 21:55:22 [error] 3299#3299: *144 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /swagger.yaml HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1", referrer: "http://localhost:3000/"
2023/02/26 21:55:22 [error] 3299#3299: *143 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /swagger.yaml HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1", referrer: "http://localhost:3000/"
2023/02/26 22:50:47 [error] 3299#3299: *199 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /swagger.yaml HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1", referrer: "http://localhost:3000/"
2023/02/26 22:50:47 [error] 3299#3299: *198 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /swagger.yaml HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1", referrer: "http://localhost:3000/"
2023/02/26 23:02:05 [error] 3299#3299: *215 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /swagger.yaml HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1", referrer: "http://localhost:3000/"
2023/02/26 23:02:05 [error] 3299#3299: *216 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /swagger.yaml HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1", referrer: "http://localhost:3000/"
2023/02/26 23:36:56 [error] 3299#3299: *279 connect() to unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /admin/barcode_blastn/nuccoresequence/b5b454fd-b05d-45a4-9e9b-1424ffb60701/change/ HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1"
2023/02/26 23:42:55 [error] 3299#3299: *281 connect() to unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /swagger.yaml HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1", referrer: "http://localhost:3000/"
2023/02/26 23:42:55 [error] 3299#3299: *281 connect() to unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /swagger.yaml HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1", referrer: "http://localhost:3000/"
2023/02/28 12:33:34 [error] 2060#2060: *28 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /swagger/ HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1"
2023/02/28 12:33:38 [error] 2060#2060: *28 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /swagger HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1"
2023/02/28 12:33:40 [error] 2060#2060: *28 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /swagger.yaml HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1"
2023/02/28 12:33:52 [error] 2060#2060: *28 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/style.css" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/style.css HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/28 12:33:52 [error] 2060#2060: *28 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/swagger-ui-dist/swagger-ui.css" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/swagger-ui-dist/swagger-ui.css HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/28 12:33:52 [error] 2060#2060: *28 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/insQ.min.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/insQ.min.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/28 12:33:52 [error] 2060#2060: *28 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/swagger-ui-init.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/swagger-ui-init.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/28 12:33:52 [error] 2060#2060: *34 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/swagger-ui-dist/swagger-ui-standalone-preset.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/swagger-ui-dist/swagger-ui-standalone-preset.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/28 12:33:52 [error] 2060#2060: *36 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/swagger-ui-dist/swagger-ui-bundle.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/swagger-ui-dist/swagger-ui-bundle.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/28 12:33:52 [error] 2060#2060: *35 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/immutable.min.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/immutable.min.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/28 12:33:52 [error] 2060#2060: *28 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/swagger-ui-dist/swagger-ui-standalone-preset.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/swagger-ui-dist/swagger-ui-standalone-preset.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/28 12:33:52 [error] 2060#2060: *28 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/insQ.min.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/insQ.min.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/28 12:33:52 [error] 2060#2060: *28 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/immutable.min.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/immutable.min.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/28 12:33:52 [error] 2060#2060: *28 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/swagger-ui-init.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/swagger-ui-init.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/28 12:33:52 [error] 2060#2060: *28 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/swagger-ui-dist/favicon-32x32.png" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/swagger-ui-dist/favicon-32x32.png HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/28 12:34:26 [error] 2060#2060: *28 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/style.css" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/style.css HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/28 12:34:26 [error] 2060#2060: *34 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/swagger-ui-dist/swagger-ui.css" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/swagger-ui-dist/swagger-ui.css HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/28 12:34:26 [error] 2060#2060: *36 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/swagger-ui-dist/swagger-ui-bundle.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/swagger-ui-dist/swagger-ui-bundle.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/28 12:34:26 [error] 2060#2060: *28 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/swagger-ui-dist/swagger-ui-standalone-preset.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/swagger-ui-dist/swagger-ui-standalone-preset.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/28 12:34:26 [error] 2060#2060: *35 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/insQ.min.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/insQ.min.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/28 12:34:26 [error] 2060#2060: *34 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/immutable.min.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/immutable.min.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/28 12:34:26 [error] 2060#2060: *36 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/swagger-ui-init.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/swagger-ui-init.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/28 12:34:26 [error] 2060#2060: *28 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/swagger-ui-dist/swagger-ui-standalone-preset.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/swagger-ui-dist/swagger-ui-standalone-preset.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/28 12:34:26 [error] 2060#2060: *28 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/insQ.min.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/insQ.min.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/28 12:34:26 [error] 2060#2060: *28 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/immutable.min.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/immutable.min.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/28 12:34:26 [error] 2060#2060: *28 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/swagger-ui-init.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/swagger-ui-init.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/28 12:34:49 [error] 2060#2060: *28 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/style.css" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/style.css HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/28 12:34:49 [error] 2060#2060: *35 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/swagger-ui-dist/swagger-ui.css" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/swagger-ui-dist/swagger-ui.css HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/28 12:34:49 [error] 2060#2060: *34 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/swagger-ui-dist/swagger-ui-bundle.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/swagger-ui-dist/swagger-ui-bundle.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/28 12:34:49 [error] 2060#2060: *28 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/swagger-ui-dist/swagger-ui-standalone-preset.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/swagger-ui-dist/swagger-ui-standalone-preset.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/28 12:34:49 [error] 2060#2060: *36 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/insQ.min.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/insQ.min.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/28 12:34:49 [error] 2060#2060: *28 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/immutable.min.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/immutable.min.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/28 12:34:49 [error] 2060#2060: *34 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/swagger-ui-init.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/swagger-ui-init.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/28 12:34:49 [error] 2060#2060: *28 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/insQ.min.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/insQ.min.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/28 12:34:49 [error] 2060#2060: *28 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/immutable.min.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/immutable.min.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/28 12:34:49 [error] 2060#2060: *28 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/swagger-ui-init.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/swagger-ui-init.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/28 12:35:10 [error] 2060#2060: *28 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/style.css" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/style.css HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/28 12:35:10 [error] 2060#2060: *35 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/swagger-ui-dist/swagger-ui.css" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/swagger-ui-dist/swagger-ui.css HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/28 12:35:10 [error] 2060#2060: *36 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/swagger-ui-dist/swagger-ui-bundle.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/swagger-ui-dist/swagger-ui-bundle.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/28 12:35:10 [error] 2060#2060: *28 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/swagger-ui-dist/swagger-ui-standalone-preset.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/swagger-ui-dist/swagger-ui-standalone-preset.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/28 12:35:10 [error] 2060#2060: *34 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/insQ.min.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/insQ.min.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/28 12:35:10 [error] 2060#2060: *28 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/immutable.min.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/immutable.min.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/28 12:35:10 [error] 2060#2060: *28 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/swagger-ui-init.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/swagger-ui-init.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/28 12:35:10 [error] 2060#2060: *28 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/swagger-ui-dist/swagger-ui-standalone-preset.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/swagger-ui-dist/swagger-ui-standalone-preset.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/28 12:35:10 [error] 2060#2060: *28 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/insQ.min.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/insQ.min.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/28 12:35:10 [error] 2060#2060: *28 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/immutable.min.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/immutable.min.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/02/28 12:35:10 [error] 2060#2060: *28 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/swagger-ui-init.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/swagger-ui-init.js HTTP/1.1", host: "127.0.0.1", referrer: "https://127.0.0.1/swagger/"
2023/03/07 21:39:18 [error] 2484#2484: *3 connect() to unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1"
2023/03/07 21:39:18 [error] 2484#2484: *3 connect() to unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1", referrer: "https://127.0.0.1/"
2023/03/07 21:39:24 [error] 2484#2484: *3 connect() to unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1", referrer: "http://localhost:3000/"
2023/03/07 21:40:03 [error] 2484#2484: *3 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1"
2023/03/07 21:54:14 [emerg] 3927#3927: unknown directive "//add_header" in /etc/nginx/sites-enabled/barcode_identifier_api_nginx.conf:50
2023/03/07 21:59:46 [emerg] 4818#4818: no "ssl_certificate" is defined for the "listen ... ssl" directive in /etc/nginx/sites-enabled/default:21
2023/03/07 21:59:58 [warn] 4857#4857: conflicting server name "127.0.0.1" on 0.0.0.0:80, ignored
2023/03/07 21:59:59 [warn] 4878#4878: conflicting server name "127.0.0.1" on 0.0.0.0:80, ignored
2023/03/07 22:00:04 [error] 4880#4880: *1 "/usr/share/nginx/html/blastdbs/index.html" is not found (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /blastdbs/ HTTP/1.1", host: "127.0.0.1"
2023/03/07 22:00:04 [error] 4880#4880: *1 open() "/usr/share/nginx/html/favicon.ico" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", host: "127.0.0.1", referrer: "http://127.0.0.1/blastdbs/"
2023/03/07 22:00:07 [error] 4880#4880: *1 open() "/usr/share/nginx/html/favicon.ico" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", host: "127.0.0.1", referrer: "http://127.0.0.1/"
2023/03/07 22:00:14 [error] 4880#4880: *1 open() "/usr/share/nginx/html/favicon.ico" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", host: "127.0.0.1", referrer: "http://127.0.0.1/"
2023/03/07 22:00:21 [error] 4880#4880: *1 "/usr/share/nginx/html/blastdbs/index.html" is not found (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /blastdbs/ HTTP/1.1", host: "127.0.0.1"
2023/03/07 22:00:21 [error] 4880#4880: *1 open() "/usr/share/nginx/html/favicon.ico" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", host: "127.0.0.1", referrer: "http://127.0.0.1/blastdbs/"
2023/03/07 22:05:36 [error] 4880#4880: *2 "/usr/share/nginx/html/blastdbs/index.html" is not found (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /blastdbs/ HTTP/1.1", host: "127.0.0.1"
2023/03/07 22:05:52 [error] 4880#4880: *2 "/usr/share/nginx/html/blastdbs/index.html" is not found (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /blastdbs/ HTTP/1.1", host: "127.0.0.1"
2023/03/07 22:06:07 [error] 4880#4880: *2 "/usr/share/nginx/html/blastdbs/index.html" is not found (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /blastdbs/ HTTP/1.1", host: "127.0.0.1"
2023/03/07 22:06:12 [error] 5558#5558: *1 "/usr/share/nginx/html/blastdbs/index.html" is not found (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /blastdbs/ HTTP/1.1", host: "127.0.0.1"
2023/03/07 22:14:37 [emerg] 6641#6641: open() "/etc/nginx/sites-enabled/default" failed (2: No such file or directory) in /etc/nginx/nginx.conf:62
2023/03/07 22:29:55 [error] 5983#5983: *42 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/style.css" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/style.css HTTP/1.1", host: "127.0.0.1:443", referrer: "http://127.0.0.1:443/swagger/"
2023/03/07 22:29:55 [error] 5983#5983: *42 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/swagger-ui-dist/swagger-ui.css" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/swagger-ui-dist/swagger-ui.css HTTP/1.1", host: "127.0.0.1:443", referrer: "http://127.0.0.1:443/swagger/"
2023/03/07 22:29:55 [error] 5983#5983: *42 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/insQ.min.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/insQ.min.js HTTP/1.1", host: "127.0.0.1:443", referrer: "http://127.0.0.1:443/swagger/"
2023/03/07 22:29:55 [error] 5983#5983: *42 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/swagger-ui-init.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/swagger-ui-init.js HTTP/1.1", host: "127.0.0.1:443", referrer: "http://127.0.0.1:443/swagger/"
2023/03/07 22:29:55 [error] 5983#5983: *48 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/swagger-ui-dist/swagger-ui-bundle.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/swagger-ui-dist/swagger-ui-bundle.js HTTP/1.1", host: "127.0.0.1:443", referrer: "http://127.0.0.1:443/swagger/"
2023/03/07 22:29:55 [error] 5983#5983: *49 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/immutable.min.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/immutable.min.js HTTP/1.1", host: "127.0.0.1:443", referrer: "http://127.0.0.1:443/swagger/"
2023/03/07 22:29:55 [error] 5983#5983: *51 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/swagger-ui-dist/swagger-ui-standalone-preset.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/swagger-ui-dist/swagger-ui-standalone-preset.js HTTP/1.1", host: "127.0.0.1:443", referrer: "http://127.0.0.1:443/swagger/"
2023/03/07 22:29:55 [error] 5983#5983: *42 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/insQ.min.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/insQ.min.js HTTP/1.1", host: "127.0.0.1:443", referrer: "http://127.0.0.1:443/swagger/"
2023/03/07 22:29:55 [error] 5983#5983: *42 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/immutable.min.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/immutable.min.js HTTP/1.1", host: "127.0.0.1:443", referrer: "http://127.0.0.1:443/swagger/"
2023/03/07 22:29:55 [error] 5983#5983: *42 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/swagger-ui-init.js" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/swagger-ui-init.js HTTP/1.1", host: "127.0.0.1:443", referrer: "http://127.0.0.1:443/swagger/"
2023/03/07 22:29:55 [error] 5983#5983: *42 open() "/home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/static/drf-yasg/swagger-ui-dist/favicon-32x32.png" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/drf-yasg/swagger-ui-dist/favicon-32x32.png HTTP/1.1", host: "127.0.0.1:443", referrer: "http://127.0.0.1:443/swagger/"
2023/03/07 22:57:06 [error] 5983#5983: *99 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /swagger/ HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1:443"
2023/03/07 23:09:33 [error] 5983#5983: *167 connect() to unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /swagger/ HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1:443"
2023/03/07 23:09:35 [error] 5983#5983: *167 connect() to unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /swagger/ HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1:443"
2023/03/07 23:09:37 [error] 5983#5983: *167 connect() to unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /swagger/ HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1:443"
2023/03/07 23:09:45 [error] 5983#5983: *167 connect() to unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /swagger/ HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1:443"
2023/03/07 23:24:03 [error] 5983#5983: *208 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /swagger/ HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1:443"
2023/03/07 23:27:33 [error] 5983#5983: *212 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /swagger/ HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1:443"
2023/03/07 23:27:43 [error] 5983#5983: *212 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /swagger/ HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1:443"
2023/03/07 23:27:45 [error] 5983#5983: *212 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /swagger/ HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1:443"
2023/03/07 23:51:55 [error] 5983#5983: *259 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /swagger/ HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1:443"
2023/03/07 23:52:30 [error] 5983#5983: *259 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /swagger/ HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1:443"
2023/03/08 00:01:02 [error] 5983#5983: *276 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /swagger/ HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1:443"
2023/03/08 00:24:43 [error] 5983#5983: *320 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /swagger/ HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1:443"
2023/03/08 00:24:46 [error] 5983#5983: *320 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /swagger/ HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1:443"
2023/03/08 00:27:21 [error] 5983#5983: *323 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /swagger/ HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1:443"
2023/03/08 00:27:44 [error] 5983#5983: *323 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /swagger/ HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1:443"
2023/03/08 00:33:35 [error] 5983#5983: *342 connect() to unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /swagger/ HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1:443"
2023/03/08 00:33:37 [error] 5983#5983: *342 connect() to unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /swagger/ HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1:443"
2023/03/08 00:33:39 [error] 5983#5983: *342 connect() to unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /swagger/ HTTP/1.1", upstream: "uwsgi://unix:///home/linux_user1/django_projects/djangorest/barcode_identifier_api/barcode_identifier_api/u.sock:", host: "127.0.0.1:443"
